dev/com.ibm.ws.sipcontainer/src/com/ibm/ws/sip/stack/transport/sip/netty/SipTcpInboundChannel.java

```diff
@@ -0,0 +1,49 @@
+/*******************************************************************************
+ * Copyright (c) 2008, 2009 IBM Corporation and others.
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v1.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-v10.html
+ *
+ * Contributors:
+ *     IBM Corporation - initial API and implementation
+ *******************************************************************************/
+package com.ibm.ws.sip.stack.transport.sip.netty;
+
+import java.io.IOException;
+import java.net.InetAddress;
+
+import com.ibm.ws.sip.stack.transaction.transport.connections.SIPConnection;
+import com.ibm.ws.sip.stack.transaction.util.SIPStackUtil;
+
+import jain.protocol.ip.sip.ListeningPoint;
+//TODO Liberty import com.ibm.ws.management.AdminHelper;
```

Comment:
Is this a todo from the original class or something we need to address here?

Reply:
deleted comment